### ISSUE TITLE
How to add text to beginning of styles?

### ISSUE THREAD

**Issue by dbrienswann** [aparajita/sketch-style-master#8]
Sorry if this is an obvious question for someone who knows regular expressions but I assume a lot of us designers do not. Is there a way to add a string of text to the beginning, or end, of all of your styles.

A really awesome addition to this plugin would some easily selectable regular expression shortcuts. :)

Really appreciate your work. Thanks for your great plugin!

**Comment by aparajita** [comment]
> Sorry if this is an obvious question for someone who knows regular expressions but I assume a lot of us designers do not

I didn't come out of the womb knowing regular expressions. 😁 At some point I read the documentation (which I linked to in the README) and experimented. There's nothing stopping designers from doing the same!

> Is there a way to add a string of text to the beginning, or end, of all of your styles.

Let's say you want to add "foo-" at the beginning of all styles:

Find: `.+`
Replace: `foo-$&`

> A really awesome addition to this plugin would some easily selectable regular expression shortcuts. :)

Regular expressions are infinite in their possibilities. As I have no way of knowing what find/replace is commonly done, as soon as I add any shortcuts someone will ask for their own personal favorites. I don't want to be in a position where I say yes to some people and no to others.

Searching or asking on stackoverflow is usually helpful to figure out how to accomplish something. Or you can search [here](https://regex101.com/library).

**Comment by dbrienswann** [comment]
Thanks for the prompt response. Wasn't expecting such a quick reply.

I did do some exploration. I ended up using:
    Find: ^ 
    Replace: "my test string"

Yeah, I know the regular expressions are infinite. I was just thinking a few quick and easy selectable examples would not only be helpful but would also encourage people to explore more of the possibilities. But I understand not wanting to go down that endless road of possibilities. 

Thanks again. I use this plugin all the time. Such a time saver.

-Brein
